Softeq.XToolkit.Common.iOS/Softeq.XToolkit.Common.iOS.csproj

```diff
@@ -22,6 +21,8 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Please set lang version and `<Nullable>` in the shared `<PropertyGroup>`, instead of build configurations (VS default behavior)